packages/react-focus/src/components/FocusZone/FocusZone.tsx

```diff
@@ -0,0 +1,1201 @@
+import * as React from 'react';
```

Comment:
Are these files just a copy of the ones in OUFR?

Reply:
yes, for now.

Reply:
This is just a scaffolding package. Once it is in a good shape, we can do a separate PR to move the other one to new location.